### DISCUSSION TITLE
How to use detekt-generator-1.23.3-all.jar?

### DISCUSSION THREAD

**Discussion by ChanningShaw** [detekt/detekt#6597] [Q&A]
Hi, when I execute the following cmd:
```
java -jar detekt-generator-1.23.3-all.jar -gcrc -i /path/to/rule1, /path/to/rule2
```
it reports:
```
detekt-generator-1.23.3-all.jar has no main manifest attribute
```
I get detekt-generator-1.23.3-all.jar from this page : https://github.com/detekt/detekt/releases
What was wrong? I need your help🙏🏻 Thank you very much!

**Comment by BraisGabin** [comment]
🤔 We should have the manifest... Can you create an issue so we can fix it?

Meanwhile I think that you can execute it with `java -cp detekt-generator-1.23.3-all.jar io.gitlab.arturbosch.detekt.generator.Main ...`

**Comment by ChanningShaw** [comment]
Sure, I have created an issue : https://github.com/detekt/detekt/issues/6600

BTW, when I execute it with 
```
java -cp detekt-generator-1.23.3-all.jar io.gitlab.arturbosch.detekt.generator.Main
```
This time it didn't report that error, but it reported another error:
```
Exception in thread "main" io.gitlab.arturbosch.detekt.generator.collection.exception.InvalidDocumentationException: 
Missing description for RuleSet extra-rules.
```
this is my RuleSet:

```
class RuleProvider : RuleSetProvider {

    override val ruleSetId: String = "extra-rules"
    override fun instance(config: Config): RuleSet = RuleSet(ruleSetId, listOf(ContextOrder(config)))
}
```

What does 'description' mean? It's very confusing. Do you have detailed usage examples or any doc about this? Otherwise, it's really difficult to use.

**Reply by BraisGabin** [reply]
No, we don't have any documentation. This is kind of new so the source is the best documentation here. Any pr to create this documentation is more than welcome.

**Reply by ChanningShaw** [reply]
ok, thanks
